### PR TITLE
Added ResonanceDecayFilter:wzAsEquivalent setting to Pythia8Interface.

### PR DESCRIPTION
**Pull request by acarvalh**
To decay H inclusivelly to a pair of gauge bosons
@kandrosov

### CODE REVIEW

**Comment by perrozzi** [comment]
please test

**Comment by perrozzi** [comment]
@acarvalh please apply the patch suggested above

File: GeneratorInterface/Pythia8Interface/src/ResonanceDecayFilterHook.cc [hunk 1]
```diff
@@ -72,12 +76,15 @@ bool ResonanceDecayFilterHook::checkVetoResonanceDecays(const Event& process) {
...
+    if( (did == 23 || did == 24) && wzAsEquivalent_) {
+      did = 23;
+    }
     
     int mid = p.mother1()>0 ? std::abs(process[p.mother1()].id()) : 0;
     
     //if no list of mothers is provided, then all particles
     //in hard process and resonance decays are counted together
-    bool validMother = mothers_.size() ? false : true;
+    bool validMother = mothers_.empty();
```
Comment (perrozzi) [inline GeneratorInterface/Pythia8Interface/src/ResonanceDecayFilterHook.cc]: are you sure you have not inverted the logic here?

Comment (perrozzi) [reply GeneratorInterface/Pythia8Interface/src/ResonanceDecayFilterHook.cc]: (probably not, just make sure)

Comment (kandrosov) [reply GeneratorInterface/Pythia8Interface/src/ResonanceDecayFilterHook.cc]: @perozzi yes, I'm sure: !size() = empt().

**Comment by acarvalh** [comment]
Dear all
Is there an prevision to when this is going to be merged?
Also, to which release we need to do the backport in order it to be included in the current prod?

thank you

**Comment by perrozzi** [comment]
please test

**Comment by perrozzi** [comment]
+1

File: GeneratorInterface/Pythia8Interface/src/ResonanceDecayFilterHook.cc (hunk 1 above)
Comment (davidlange6) [reply GeneratorInterface/Pythia8Interface/src/ResonanceDecayFilterHook.cc]: hi @acarvalh - if this line is correct, then the loop just below is not needed. perhaps it should really be
bool validMother=false?[in fact the declaration of validMother can be avoided trivially]

**Comment by perrozzi** [comment]
please test

**Comment by perrozzi** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**